addons/edi/views/edi_menu_views.xml

```diff
@@ -13,6 +13,10 @@
     <menuitem name="Records" id="record_menu" parent="root_menu"
 	      sequence="20"/>
 
+    <!-- Records/UDES menu -->
+    <menuitem id="record_udes_menu" name="UDES"
```

Comment:
It was originally in `udes_stock`, but I moved it into EDI because it felt wrong that it was defined in `udes_stock` as it has nothing to do with the products? Maybe I should move it into `udes_interfaces`/`udes_rockhopper` instead?

Reply:
As discussed, let's move this to the module where we'll add UDES EDI records. :)